app/views/state_file/questions/w2/edit.html.erb

```diff
@@ -9,14 +9,24 @@
       <p class="spacing-below-25"><%= @w2.employer_name %></p>
 
       <% if @box14_codes.present? %>
+        <% box14_intro_key = "state_file.questions.w2.edit.box14_intro_#{current_state_code}_html" %>
+        <% if I18n.exists?(box14_intro_key) %>
+          <p><%= t(box14_intro_key) %></p>
+        <% end %>
         <fieldset>
           <legend class="spacing-below-5"><%= t(".box14_html") %></legend>
           <% @box14_codes.each do |code| %>
             <div class="form-question spacing-below-25">
               <% field_name = "box14_#{code.downcase}" %>
+              <% value = if code == "UIWFSWF"
+                @w2.get_box14_ui_overwrite
+              else
+                @w2.send(field_name)
+              end %>
```

Comment:
review comment: could a method on the W2 model prevent us from having to do all these if-else to get these values?

Reply:
@mmazanec22 I want to keep this one open too, but since it's only used in a few spots (and in different ways) I think I want to leave as a follow-up